### DISCUSSION TITLE
Why we need the FrameworkElement.Language property?

### DISCUSSION THREAD

**Discussion by yll690** [dotnet/wpf#9437] [Q&A]
Just for curious. Since we already have CurrentCulture and CurrentUICulture, why we need the FrameworkElement.Language? I think it brings more trouble as it has an inappropriate default value and need to be set separately.

**Comment by miloush** [comment]
Because the language specifies the language of the content.

> The language attribute is very useful for specifying regional languages. For example, French has different spelling, vocabulary, and pronunciation in France, Quebec, Belgium, and Switzerland. Also Chinese, Japanese, and Korean share code points in Unicode, but the ideographic shapes are different and they use totally different fonts.

**Reply by yll690** [reply]
Yes you are right. But what I mean is since CurrentCulture and CurrentUICulture have done the same thing, why we need an extra FrameworkElement.Language property instead of using CurrentCulture and CurrentUICulture.
